src/requests/interface.js

```
@@ -44,6 +45,36 @@ const browseExistingList = function (resourceType = '', organization_code = null
+					isObservable: true,
+				},
+				projection: ['_id', 'name'],
+			}
+
+			const apiUrl = utils.buildUrl(entityManagementBaseUrl, endpoints.ENTITY_TYPES_READ)
+			const result = await requests.post(apiUrl, data, '', true, 'internal-access-token')
+			return resolve(result)
+		} catch (error) {
+			return reject(error)
+		}
+	})
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)